doc/plot.md

```diff
@@ -45,15 +65,15 @@ Using libraries like `matplotlib` or `seaborn`, requires fetching all the data l
 We'll be using a sample dataset that contains information on music tracks:
 
 ```{code-cell} ipython3
-%%sql
-SELECT * FROM "TrackAll" LIMIT 2
+%%sql sqlite:///my.db
+SELECT * FROM "Track" LIMIT 3
 ```
 
-The `TrackAll` table contains 2.9 million rows:
+The `Track` table contains 3503 rows:
```

Comment:
i think there's a table that has a lot more rows, we should use that one. the idea of this example is to show how to plot large datasets efficiently, so using such a small table defeats the purpose